sample/tests.yml

```diff
@@ -3,4 +3,8 @@ general:
   - "./general/jobs/"
   jobs:
     test-pipeline: 
-      job: "./general/jobs/pipeline.yml"
\ No newline at end of file
+      job: "./general/jobs/pipeline.yml"
+    sample_pytest: 
+      pytest_job: "./general/jobs/sample_pytest.py"
+      assets:
+      - sample_component
```

Comment:
can we change this to point to asset file? There are multiple issues with this approach you have right now:
1. assets may have dup names
2. assets of different types may have dup names (e.g. env and component)
I initially was thinking to add type here, but I feel pointing to the actual asset spec file is the most accurate.